test/test_ops.py

```diff
@@ -1577,6 +1577,121 @@ def is_bit_set(x):
             torch.is_complex,
         )
 
+    @onlyCUDA
+    def test_opinfo_aten_ops_coverage(self):
+        all_aten_ops = {
+            torch.ops.aten.__getattr__(attr)
+            for attr in dir(torch.ops.aten)
+            if not attr.startswith('__') and attr not in {'name'} and
+            isinstance(torch.ops.aten.__getattr__(attr), torch._ops.OpOverloadPacket)
+        }
+
+        # Copies inputs to inplace operations to avoid inplace modifications
+        #   to leaves requiring gradient
+        def _get_safe_inplace(inplace_variant):
```

Comment:
Let's maybe refactor this to be a helper in `common_utils.py`? There's already another definition there:

https://github.com/pytorch/pytorch/blob/c407a7b20330afb957944ad26633a388220a4e43/torch/testing/_internal/common_utils.py#L3617